spec/models/member_spec.rb

```diff
@@ -160,4 +163,165 @@
       it { is_expected.not_to be_thredded_admin }
     end
   end
+
+	describe "renewed_subscription" do
+
+			before(:each) do
+				@member = FactoryBot.create(:member)
+			end
```

Comment:
You can replace this `before` setup with `let(:member) { create :member }`
This will scope the `member` variable to the current example block, instead of creating a `@member` instance variable that can be accessed from anywhere in the class.

FactoryBot is also included directly in `spec_helper.rb`, so you can omit `FactoryBot.` when calling FactoryBot methods